final_prep/mod_zero_hero.rb

```diff
@@ -74,11 +116,40 @@
 
 # - Create 2 instances of your SuperHero class
 
+class SuperHero
+
+  def initialize(name, super_power, age)
+    @name = name
+    @super_power = super_power
+    @age = age
+  end
+
+  def initalize(arch_nemesis, power_level, energy_level)
+    @ARCH_NEMESIS = "The Syntax Error"
```

Comment:
Make sure you're using lowercase letters here, so @ARCH_NEMESIS should be @arch_nemisis